### ISSUE TITLE
5.4.1 Non observed properties are no longer included

### ISSUE THREAD

**Issue by timothyallan** [mobxjs/mobx-utils#205]
So 5.4.1 fixed a previous error for https://github.com/mobxjs/mobx-utils/issues/191 but is causing another. Any properties that aren't marked as observable simply don't appear anymore.

If you cut and paste the below code into the demo sandbox, you'll see what happens. the `notObservedField` doesn't exist... and in 5.2 it did.

https://codesandbox.io/s/wp88p

```
import React from "react";
import { render } from "react-dom";
import { observable, action, toJS } from "mobx";
import { observer } from "mobx-react";
import DevTools from "mobx-react-devtools";
import { createViewModel } from 'mobx-utils';

class MyClass {
  @observable firstname;
  @observable lastname;
  notObservedField = 'im not included!';
  someFunction(someParam) {
  }
}

class AppState {
  constructor() {
    const myClass = new MyClass();
    const observableMyClass = createViewModel(myClass);
    console.log(toJS(observableMyClass));
    // is now undefined in 5.4.1
    console.log(observableMyClass.notObservedField);
  }

  @action.bound
  reset() {
    this.timer = 0;
  }
}

const TimerView = observer(({ appState }) => (
  <button onClick={appState.reset}>Seconds passed: {appState.timer}</button>
));


render(
  <div>
    <TimerView appState={new AppState()} />
    <DevTools />
  </div>,
  document.getElementById("root")
);
```

Edit: I should also mention that this only happened in production builds for me? Dev builds worked fine. Was a bit of a panic after rollover.

**Comment by ItamarShDev** [comment]
`notObservedField` was not logged also in 5.4.0 and 5.3.0
Am i missing the bugs? 

https://codesandbox.io/embed/minimal-mobx-react-project-jh2lz

**Comment by timothyallan** [comment]
it worked in 5.2 which was the version we upgraded from.

This is a very breaking change for our app having all those fields just vanish. Shouldn’t it be a major version bump if it’s not a bug?

**Comment by urugator** [comment]
The [`getAllMethodsAndProperties`](https://github.com/mobxjs/mobx-utils/blob/master/src/utils.ts#L35) seems to be missing the *"Properties"* part, there are just getters and functions right?

**Comment by jasonlangdon-sky** [comment]
@urugator right you are. That was an oversight on my behalf as the model I was passing to createViewModel only had observable properties, and no unobserved ones. I'm working on a fix and hope to have it on a branch today.

**Comment by mweststrate** [comment]
Published as 5.5.0
